src/test/java/teammates/test/cases/action/InstructorEditStudentFeedbackSaveActionTest.java

```diff
@@ -77,11 +77,13 @@ private void testModifyResponses() {
 
         assertFalse(r.isError);
         assertEquals(Const.StatusMessages.FEEDBACK_RESPONSES_SAVED, r.getStatusMessage());
-        assertEquals("/page/instructorEditStudentFeedbackPage"
-                     + "?error=false&moderatedperson=student1InIESFPTCourse%40gmail.tmt"
-                     + "&user=IESFPTCourseinstr&courseid=IESFPTCourse"
-                     + "&fsname=First+feedback+session",
-                     r.getDestinationWithParams());
+        assertEquals(
+                getPageResultDestination(
+                        Const.ActionURIs.INSTRUCTOR_EDIT_STUDENT_FEEDBACK_PAGE,
+                        false, "student1InIESFPTCourse%40gmail.tmt",
+                        "IESFPTCourseinstr", "IESFPTCourse",
+                        "First+feedback+session"),
```

Comment:
Here's another case. Either give one parameter for line, or pack more parameters in one line.